client/src/components/Nav/Logout.jsx

```diff
@@ -20,8 +20,7 @@ const Logout = forwardRef(() => {
       onClick={handleLogout}
     >
       <LogOutIcon />
-      {user?.username || localize(lang, 'com_nav_user')}
-      <small>{localize(lang, 'com_nav_log_out')}</small>
+      {localize(lang, 'com_nav_log_out')}
```

Comment:
refactor this component to TSX and utilize useLocalize hook. CTRL+F for examples, there are many.

Reply:
@danny-avila Fixed!